File: app/src/main/java/com/ravisharma/playbackmusic/database/SetupDao.java
package com.ravisharma.playbackmusic.database;

import androidx.lifecycle.LiveData;
import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;
import com.ravisharma.playbackmusic.database.model.DatabaseSetup;

@Dao
public interface SetupDao {

    @Insert
    void setUp(DatabaseSetup setup);


    @Query("SELECT * FROM setupTable")
    LiveData<DatabaseSetup> dataSetup();
}

File: app/src/main/java/com/ravisharma/playbackmusic/database/dao/SetupDao.kt
package com.ravisharma.playbackmusic.database.dao

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query
import com.ravisharma.playbackmusic.database.model.DatabaseSetup

@Dao
interface SetupDao {
    @Insert
    fun setUp(setup: DatabaseSetup)

    @Query("SELECT * FROM setupTable")
    fun dataSetup(): LiveData<DatabaseSetup>
}